### DISCUSSION TITLE
Audio / Video inconsistencies

### DISCUSSION THREAD

**Discussion by zach-vz** [mozilla/hubs#2784] [General Questions & Discussion] [answered]
Hi all,

Not sure if Github issues is the right place for this, Discussions, or both. I posted in issues (#2643) and others have noted it as well, particularly in the last 24 hours though we've been experiencing it for at least a week.

Some users are experiencing an issue where they are not able to be seen / heard by other users in the space. This seems to happen consistently and with some regularity without pattern to who the users are, what machine types or browsers they are on (though most are on a very recent version of Chrome), etc. The only pattern we've *maybe* noticed is that using a custom avatar could be related as we've experienced it, at least, more frequently with custom avatars than with the avatars included with Hubs. This includes avatars made with readyplayer.me.

Have you noticed this internally?

Thanks!

Zach

**Comment by brianpeiris** [comment, marked as answer]
Hi @zach-vz, if you hadn't noticed, we're collecting information from users to help pin this issue down, as mentioned in #2643: https://github.com/mozilla/hubs/issues/2643#issuecomment-669766510

**Comment by zach-vz** [comment]
Yep, I’m one of them Brian!

On Mon, Aug 10, 2020 at 4:29 PM Brian Peiris <notifications@github.com>
wrote:

> Hi @zach-vz
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_zach-2Dvz&d=DwMFaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=4ftBV0YVSJjhg1ZhVWzxUU9P3ZCH-bQhOsh1WxKF9lA&m=FWlihYa55xiSAiIrAOi2hGl0uMrdgXqZRvwFDBOA0SI&s=yYDOq7NW6Sp7TOK7O9-FTWFY6xUnEIIQNTeluIrnQCw&e=>,
> if you hadn't noticed, we're collecting information from users to help pin
> this issue down, as mentioned in #2643
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mozilla_hubs_issues_2643&d=DwMFaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=4ftBV0YVSJjhg1ZhVWzxUU9P3ZCH-bQhOsh1WxKF9lA&m=FWlihYa55xiSAiIrAOi2hGl0uMrdgXqZRvwFDBOA0SI&s=biKTDPc1Q9wD5yRf6tnZSzxOtjfWTeM8UhYghid9SOE&e=>:
> #2643 (comment)
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mozilla_hubs_issues_2643-23issuecomment-2D669766510&d=DwMFaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=4ftBV0YVSJjhg1ZhVWzxUU9P3ZCH-bQhOsh1WxKF9lA&m=FWlihYa55xiSAiIrAOi2hGl0uMrdgXqZRvwFDBOA0SI&s=n4F3CdM436b4A-xYL55_41pQO0wmeYUgKCEzKLmKmZI&e=>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mozilla_hubs_discussions_2784-23discussioncomment-2D49248&d=DwMFaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=4ftBV0YVSJjhg1ZhVWzxUU9P3ZCH-bQhOsh1WxKF9lA&m=FWlihYa55xiSAiIrAOi2hGl0uMrdgXqZRvwFDBOA0SI&s=lNWXL6sqIdQjbPbch8cijhmDew95_X-sN_z0JuggCw0&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AMI2I4S765BJDQTQHIWNXI3SABKA5ANCNFSM4PUN37VQ&d=DwMFaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=4ftBV0YVSJjhg1ZhVWzxUU9P3ZCH-bQhOsh1WxKF9lA&m=FWlihYa55xiSAiIrAOi2hGl0uMrdgXqZRvwFDBOA0SI&s=WxZvdpsH6L8DjaOlx4lWMoYa41460v51EaQf3El4TMI&e=>
> .
>
-- 
<http://www.verizon.com>

Zachary Drossman

Head of Concept Development
5G Labs

M 347 963 7237
119 W24th St
5G Labs at Alley Chelsea
New York, NY 10011
